rskj-core/src/main/java/co/rsk/peg/UnauthorizedVoterException.java

```diff
@@ -0,0 +1,7 @@
+package co.rsk.peg;
+
+public class UnauthorizedVoterException extends RuntimeException {
```

Comment:
Maybe this should go in `abi` package?

Reply:
i suggest we create an `/authorizer` package inside `/peg`, to have this class and AdressBasedAuthorizer. 
Wdyt?
This way we avoid the circular dependency between `/peg` and `/peg/abi` too